applications/machine_learning/remote/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@
 
 cmake_minimum_required(VERSION 3.20.0)
 
-if (NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}")
+if (NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/configuration/${NORMALIZED_BOARD_TARGET}")
```

Comment:
you don't have `NORMALIZED_BOARD_TARGET` available at this point in time, so this will not work as expected.

Reply:
Thanks. fixed